### ISSUE TITLE
Non-tested namespace declaration

### ISSUE THREAD

**Issue by rimace** [sebastianbergmann/php-code-coverage#723]
| Q                         | A
| --------------------------| ---------------
| php-code-coverage version | 8.0.0
| PHP version               | 7.4.2 or 7.3.14
| Driver                    | Xdebug
| Xdebug version (if used)  | 2.9.1
| Installation Method       | Composer
| Usage Method              | PHPUnit
| PHPUnit version (if used) | 9.0.0

Since upgrading to PHPUnit 9.0 and php-code-coverage 8.0 the coverage report reports less lines are tested.
According to the coverage report the first line with `<?php` and the third lines (which contains only the namespace) is not covered by tests.

```
<?php

namespace App\Contracts;

interface UsesGeneratableId
{
	/**
	 * Create the ID for this object.
	 *
	 * @return void
	 */
	public function recreateId(): void;
}
```
Some interfaces and classes work as expected and are fully covered. Others are not because the line with the namespace is not covered by tests.

**Comment by sebastianbergmann** [comment]
Thank you for your report.

Please provide a **minimal**, **self-contained**, reproducing test case that shows the problem you are reporting.

Without such a minimal, self-contained, reproducing test case I will not be able to investigate this issue.

**Comment by someniatko** [comment]
@sebastianbergmann https://github.com/someniatko/phpunit9-coverage-bug

**Comment by sebastianbergmann** [comment]
That repository has no test and no PHPUnit configuration file.

**Comment by someniatko** [comment]
@sebastianbergmann 
Firstly, adding a test is redundant for the bug reproduction. The issue is that 3 lines were counted as "uncovered" on phpunit 9.0, while on phpunit 8.5 no lines are marked uncovered.

Secondly, there is `phpunit.xml.dist` file.

**Comment by sebastianbergmann** [comment]
`--coverage-text` is useless, but I can reproduce the issue with `--coverage-html`:

![Screenshot from 2020-02-12 14-25-15](https://user-images.githubusercontent.com/25218/74339112-b337ad80-4da3-11ea-8641-e4b2a521ac9c.png)

My apologies for overlooking `phpunit.xml.dist` earlier.

**Comment by someniatko** [comment]
Yeah, it's a bit useless, however you can still check the numbers: `0/3` vs `0/0` :)

**Comment by sebastianbergmann** [comment]
This is not a general problem, as can be seen in the HTML code coverage report for PHPUnit itself:
![Screenshot from 2020-02-12 14-29-20](https://user-images.githubusercontent.com/25218/74339386-42dd5c00-4da4-11ea-88ef-f41af664e1a3.png)

Not sure what is going on here, could be related to no tests being run.

**Comment by someniatko** [comment]
@sebastianbergmann i have the same behavior in a real project i am working on, with tests of course. I suspect it happens if the given interface is the **first file** checked in the directory.

**Comment by someniatko** [comment]
Looks like it was fixed in PHPUnit 9.0.1

**Comment by sebastianbergmann** [comment]
Ah, so it was a side effect of https://github.com/sebastianbergmann/phpunit/issues/4036.
